source/extensions/clusters/redis/redis_cluster_lb.h

```diff
@@ -50,7 +50,7 @@ class ClusterSlot {
   int64_t start_;
   int64_t end_;
   Network::Address::InstanceConstSharedPtr primary_;
-  absl::flat_hash_set<Network::Address::InstanceConstSharedPtr> replicas_;
+  std::map<std::string, Network::Address::InstanceConstSharedPtr> replicas_;
```

Comment:
I think normally we perfer to use the absl maps, one of `absl::flat_hash_map`, `absl::node_hash_map` if you want pointer stability or `absl::btree_map` if you want an ordered map.

Reply:
I have changed it to absl::btree_map and created a new commit.